app/assets/stylesheets/components/typeahead.scss

```diff
@@ -1,7 +1,46 @@
+.tt-menu {
```

Comment:
You are right. I made 2 or 3 changes due upgrade and previous enhancement but I'll keep it in vendor folder. Thanks for reminding me that. :)